.gitignore

```diff
@@ -1,3 +1,7 @@
-.DS_Store
+.idea/
+venv/
 
-yolov3.weights
\ No newline at end of file
+.DS_Store
+yolov3.weights
+yolov3.cfg
+coco.names
```

Comment:
@guinnesschen I think if you're not pushing yolov3.weights, then we should do the same to yolov3.cfg & coco.names, since they are model-specific.